Chutzpah/TestRunner.cs

```diff
@@ -305,10 +305,15 @@ private IChutzpahWebServerHost SetupWebServerHost(ConcurrentBag<TestContext> tes
             if (contextWithChosenServerConfiguration != null)
             {
                 var webServerConfiguration = contextWithChosenServerConfiguration.TestFileSettings.Server;
-                webServerHost = webServerFactory.CreateServer(webServerConfiguration, ActiveWebServerHost);
 
-                // Stash host object on context for use in url generation
-                contextUsingWebServer.ForEach(x => x.WebServerHost = webServerHost);
+                // create host if it no value (default) or explicit enabled.
+                if (!webServerConfiguration.Enabled.HasValue || webServerConfiguration.Enabled.Value == true)

```

Comment:
I don't undertand why we can't just check if it is enabled == true since in chutzpahcontextsettings service we set it to true in this case no?

Reply:
Good point. Fixed